test/test.cpp

```diff
@@ -222,24 +243,48 @@ TEST(nogo, server)
         // fmt::print("\033[32mrecv_msg2: {}\033[0m\n", recv_msg2);
 
         constexpr string_view placeholder = "{TIMEOUT}";
-        constexpr auto timestamp_len = 10;
+        constexpr auto timestamp_len = 13;
```

Comment:
why timestamp len equals 13?

Reply:
> why timestamp len equals 13?

https://github.com/The-Goo-Goo-Gang/nogo-backend/commit/b29f5d33317b10a30e7449d719808e88675835db